job/server/src/main/java/alluxio/master/job/JobMasterClientRestServiceHandler.java

```diff
@@ -110,7 +107,6 @@ public Void call() throws Exception {
    */
   @GET
   @Path(ServiceConstants.GET_STATUS)
-  @JacksonFeatures(serializationEnable = {SerializationFeature.INDENT_OUTPUT})
```

Comment:
Removing this is fine?

Reply:
It's merely for formatting - and none of the other calls format their output, so I assumed removing this is ok.